From 39e39d6510ef2d7a8d5b8a589672987da61f32c9 Mon Sep 17 00:00:00 2001
From: mbah37 <mbah37@student.cscc.edu>
Date: Tue, 11 Nov 2025 23:48:51 -0500
Subject: [PATCH 02/17] 1st tutorial. Created game loop and set up display for
 the game!

---
 alien_invasion.py | 29 ++++++++++++++++++++++++++++-
 1 file changed, 28 insertions(+), 1 deletion(-)

diff --git a/alien_invasion.py b/alien_invasion.py
index 0061e9f..94d5dc1 100644
--- a/alien_invasion.py
+++ b/alien_invasion.py
@@ -1,5 +1,32 @@
 import sys
 import pygame
 
+
+class AlienInvasion:
+
+    def __init__(self):
+        pygame.init()
+
+        self.screen = pygame.display.set_mode((1200, 800))
+        pygame.display.set_caption("Alien Invasion")
+
+        self.running = True
+    
+    def run_game(self):
+        while self.running:
+            for event in pygame.event.get():
+                if event.type == pygame.QUIT:
+                    self.running = False
+                    pygame.quit()
+                    sys.exit()
+            
+            pygame.display.flip()
+
+
+
+
+
+
 if __name__ == '__main__':
-    pass
+    ai = AlienInvasion()
+    ai.run_game()
\ No newline at end of file
-- 
2.51.0.windows.1

