From 1912d2d43a393e615ddffeabe5cdfcf775805efd Mon Sep 17 00:00:00 2001
From: mbah37 <mbah37@student.cscc.edu>
Date: Tue, 11 Nov 2025 23:50:17 -0500
Subject: [PATCH 03/17] deleted extra space

---
 alien_invasion.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/alien_invasion.py b/alien_invasion.py
index 94d5dc1..9972627 100644
--- a/alien_invasion.py
+++ b/alien_invasion.py
@@ -22,11 +22,6 @@ class AlienInvasion:
             
             pygame.display.flip()
 
-
-
-
-
-
 if __name__ == '__main__':
     ai = AlienInvasion()
     ai.run_game()
\ No newline at end of file
-- 
2.51.0.windows.1

