From b3eaa2371da52da5687614f819121532d836d0d5 Mon Sep 17 00:00:00 2001
From: mbah37 <mbah37@student.cscc.edu>
Date: Wed, 12 Nov 2025 15:11:01 -0500
Subject: [PATCH 07/17] change alien invasion class to whitewalker invasion

---
 alien_invasion.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/alien_invasion.py b/alien_invasion.py
index 498800f..07b8f8d 100644
--- a/alien_invasion.py
+++ b/alien_invasion.py
@@ -2,7 +2,7 @@ import sys
 import pygame
 from settings import Settings
 
-class AlienInvasion:
+class WhiteWalkerInvasion:
 
     def __init__(self):
         pygame.init()
@@ -32,5 +32,5 @@ class AlienInvasion:
             self.clock.tick(self.settings.FPS)
 
 if __name__ == '__main__':
-    ai = AlienInvasion()
+    ai = WhiteWalkerInvasion()
     ai.run_game()
\ No newline at end of file
-- 
2.51.0.windows.1

