From 600688df0d0f6920ddc48357220d61d6b8776e34 Mon Sep 17 00:00:00 2001
From: mbah37 <mbah37@student.cscc.edu>
Date: Wed, 12 Nov 2025 16:27:18 -0500
Subject: [PATCH 09/17] 4th tutorial. Refactored game loop for events and
 screen.

---
 alien_invasion.py | 25 ++++++++++++++++---------
 1 file changed, 16 insertions(+), 9 deletions(-)

diff --git a/alien_invasion.py b/alien_invasion.py
index d1135e9..020187b 100644
--- a/alien_invasion.py
+++ b/alien_invasion.py
@@ -26,17 +26,24 @@ class WhiteWalkerInvasion:
     
     def run_game(self):
         while self.running:
-            for event in pygame.event.get():
-                if event.type == pygame.QUIT:
-                    self.running = False
-                    pygame.quit()
-                    sys.exit()
-            self.screen.blit(self.bg, (0, 0))
-            # change variable to dragon. remove comment when done
-            self.ship.draw()
-            pygame.display.flip()
+            self._check_events()
+            
+            self._update_screen()
             self.clock.tick(self.settings.FPS)
 
+    def _update_screen(self):
+        self.screen.blit(self.bg, (0, 0))
+        # change variable to dragon. remove comment when done
+        self.ship.draw()
+        pygame.display.flip()
+
+    def _check_events(self):
+        for event in pygame.event.get():
+            if event.type == pygame.QUIT:
+                self.running = False
+                pygame.quit()
+                sys.exit()
+
 if __name__ == '__main__':
     ai = WhiteWalkerInvasion()
     ai.run_game()
\ No newline at end of file
-- 
2.51.0.windows.1

